docs/changelogs/vNEXT.rst

```diff
@@ -28,6 +28,16 @@ authorization instead of the removed
   extra positional arguments. Use keyword arguments to pass options to the
   session.
 
+
+Removed "extras"
+----------------
+
+The oso library previously had some additional default supported classes:
+``Http`` and ``Pathmapper``. These have been deprecated and removed.
+
+Instead, you should register suitable classes directly. For example,
+registering the ``werkzeug.wrappers.Request`` object for Flask applications.
```

Comment:
Looks like we already (conditionally) register the `flask.Request` class (a subclass of `werkzeug.wrappers.Request`) automatically in the `flask-oso` lib.

Reply:
Hmm, that's a good point. One of the reasons for removing Http is that you probably just want whatever the framework uses.

Reply:
Updated.